packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -128,7 +128,9 @@ const joinItems = (form: OrderForm) => {
       const [item] = items
       const quantity = items.reduce((acc, i) => acc + i.quantity, 0)
       const totalPrice = items.reduce(
-        (acc, i) => acc + i.quantity * i.sellingPrice,
+        (acc, i) =>
+          acc +
+          (i?.priceDefinition?.total ?? i?.quantity * i?.sellingPrice ?? 0),
```

Comment:
Not sure if this scenario is possible, but this expression can return `NaN` if `i?.priceDefinition?.total` and (`i?.quantity` OR `i?.sellingPrice`) are `undefined`:
<img width="243" alt="Screenshot 2023-09-04 at 18 08 28" src="https://github.com/vtex/faststore/assets/11325562/9400ff6b-a85c-4029-9359-f632ef1ac74a">

what about:
```suggestion
          (i?.priceDefinition?.total ?? (i?.quantity ?? 0) * (i?.sellingPrice ?? 0)),
```

Reply:
looks better to me and more readable :) thanks